loomi/regulariza-morador-client
# Feature/research inputs

## Feature
- Date Input
- Number Input
- Select Input

**What I did:**

- Add masks in text input
- add preview in file input
- add currency mask in input number
- fix some questions in db mock

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/133142813-4b4877d5-04fb-4098-89c0-4ba856bfbe5c.png)
![image](https://user-images.githubusercontent.com/32523401/133142854-409f1b92-abd8-4504-b7e7-1df9ad6c13c0.png)
![image](https://user-images.githubusercontent.com/32523401/133142953-4119afcd-c5a1-432f-bf47-a4e613009ced.png)
